plutus-use-cases/src/Language/PlutusTx/Validation.hs

```diff
@@ -1,80 +0,0 @@
-module Language.PlutusTx.Validation (
-    module TH
-    ) where
-
-import           Language.PlutusTx.Validation.TH as TH
-
-{- Note [Script types in pending transactions]
```

Comment:
These notes seem to have been lost, not sure if you want to put them elsewhere?

Reply:
done. I moved them to their new homes and updated them.